contact/style.scss

```
@@ -0,0 +1,143 @@
+body{
+  background-color: #FFFFFF;
+  margin: 36px;
+}
+
+h1{
+  height: 22px;
+  text-align: right;
+  hirgt:54px;
+}
+
+.content-page-title{
+  color: #000000;
+  text-align: center;
+  hirgt:77px;
+}
+
+
+
+
+
+.contact-primary{
+  p{
+    color: #707070;  
+    text-align: center;
+    height: 107px;
+    margin: 36px;
+    }
+
+  ul{
+  display: block;
+  list-style: none;
+  font-size: 2.0rem;
+  /*1.0remのフォントサイズを確認するにはどうすればよいか、ブラウザ依存？*/
+  height: 45px;
+  width: 171px;
+  }
+
+  button{
+    outline:0;
+  }
+  
+  img{
+    height: 31px;
+    width: 22px;
+  }
+
+
+
+}
+
+
+
+
+
+.contact-tpl{
+  &__description{
+      color: #707070;
+      text-align: center; 
+  }
+  &__button-1st_wrapper{
+    text-align:center; 
+  }
+  &__button-1st{ 
+      color: #707070;
+      border-radius:98;
```

```suggestion
      border-radius:98;
```
border-radius は、単位が必要。 px など

---

問題ありません